Nos/Views/Settings/DeleteConfirmationView.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+
+/// This view displays a title and message, along with a text field
+/// for user confirmation. It provides buttons to cancel the action
+/// or proceed with the deletion (which is enabled only when the
+/// correct confirmation text is entered).
+/// - Parameters:
+///   - requiredText: The text that must be entered to confirm the deletion.
+///   - onDelete: A closure that is called when the delete button is confirmed.
+///   - onCancel: A closure that is called when the cancel button is selected.
+struct DeleteConfirmationView: View {
+    @State private var confirmationText: String = ""
+
+    var requiredText: String
+    var onDelete: (() -> Void)
+    var onCancel: (() -> Void)
+
+    var isDeleteDisabled: Bool {
+        confirmationText != requiredText
+    }
+
+    var body: some View {
+        VStack(spacing: 0) {
+            VStack(spacing: 0) {
+                titleMessageView()
+
+                confirmationTextField(
+                    confirmationText: $confirmationText,
+                    requiredText: requiredText
+                )
+            }
+            .padding()
+
+            Divider()
+                .frame(height: 1)
+                .background(Color.actionSheetDivider)
+
+            actionButtonsView(
+                isDeleteDisabled: isDeleteDisabled, 
+                onDelete: onDelete,
+                onCancel: onCancel
+            )
+        }
+        .background(Color.actionSheetBg)
+        .cornerRadius(10)
+        .frame(width: 302, height: 228)
+        .shadow(color: Color.black.opacity(0.25), radius: 25, x: 0, y: 10)
+    }
+
+    /// Creates the title and message view for the delete confirmation.
+    private func titleMessageView() -> some View {
+        VStack(spacing: 0) {
+            Text(String(localized: "deleteAccount"))
+                .font(.headline)
+                .fontWeight(.bold)
+                .padding(.bottom, 10)
+                .padding(.top, 18)
+
+            Text(String(localized: "deleteAccountMessage"))
+                .font(.footnote)
+                .fixedSize(horizontal: false, vertical: true)
+                .multilineTextAlignment(.center)
+                .foregroundColor(.white)
+                .padding(.horizontal)
+                .padding(.bottom, 15)
+        }
+    }
+
+    /// Creates the text field to confirm the user's input.
+    /// - Parameters:
+    ///   - confirmationText: A binding to the user's input text.
+    ///   - requiredText: The text that must be entered for confirmation.
+    /// - Returns: A `TextField` configured for confirming user's input.
+    private func confirmationTextField(confirmationText: Binding<String>, requiredText: String) -> some View {
+        let isTyping = !confirmationText.wrappedValue.isEmpty
+
+        return TextField("", text: confirmationText)
```

Comment:
Why did you decide to go with a PlaceholderStyle modifier instead of using `TextField`'s built-in placeholder?

Reply:
I needed to change the placeholder text color to match what is on figma.